### DISCUSSION TITLE
How did our organization get on the r-universe build list?

### DISCUSSION THREAD

**Discussion by martinamorris** [r-universe-org/help#78] [Q&A] [answered]
Hello out there ...  This seems like a very quiet discussion site, so, fingers crossed.

Our GH organization, [statnet](https://github.com/statnet) was added to the r-universe list -- which is great, btw -- but we're not sure how that happened.

We don't have a `universe` repo, and we're not sure why some of our repos have been added, but others not (e.g., [tergmLite](https://github.com/statnet/tergmLite)).

We'd love to know more about how to maintain/update our stuff on r-universe.  Thx!

I'm guessing @jeroen might know?

**Comment by jeroen** [comment, marked as answer]
statnet was one of the lucky organizations that I manually enabled to test the r-universe system :-)

If you don't have a universe repo, the packages are scraped from CRAN. We look for all CRAN packages that have a URL or BugReports field that points to `https://github.com/statnet/*`.  You can find the current list here, it gets updated daily: https://github.com/r-universe-org/cran-to-git/blob/master/statnet.json

Once you create a `universe` repo, this will take precedence an you control yourself which packages get included.

This is also described in the last section in this blog post: https://ropensci.org/blog/2021/06/22/setup-runiverse/#auto-generated-universes-for-cran-developers

**Reply by martinamorris** [reply]
And we *thank you* for making us a guinea pig.  We've made use of your builds already several times in a pinch when we discover a bug just before a workshop, and need the binaries for participants in short order.  This is a really useful system you've set up.

Thanks, too, for the quick response.
